### ISSUE TITLE
Info is not saving

### ISSUE THREAD

**Issue by IvanMMM** [micovery/ArmA3_Wasteland.Altis#18]
Looks like info from "o_saveInfo" function is not saving. Data is pushed to request correctly, but there's no entry at database.

**Comment by IvanMMM** [comment]
It's not saving only after migration. If db is new, it's working fine.
DB before first server launch: http://prntscr.com/52l75j
DB after first server launch: http://prntscr.com/52l72a

**Comment by IvanMMM** [comment]
And after some time it's removed again.
http://prntscr.com/52l7h2

**Comment by micovery** [comment]
Yeah, I noticed too, fixed it on a0a3c19c72a0eb195144b5e68fe7898ae6b1b4a6

See 

https://github.com/micovery/ArmA3_Wasteland.Altis/blame/Development_main/persistence/players/sFunctions.sqf#L36

**Comment by IvanMMM** [comment]
I'v installed fix, but still nothing.

**Comment by micovery** [comment]
There is got to be some other problem going on, from your screenshot it appears that the `stats_wipe` on object happens, but then the `o_saveInfo` call never happens. Can you run the stats server with `--trace-req` and, and then force a stats save by using the `restart-server-hint` script.  You should see  something like this

```
1. The "Altis_Objects" document is wiped
  { "method": "wipe", "params": ["Altis_Objects"]}

2. The current list of objects in the map is saved to the "Altis_Objects" document
  { "method": "set", "params": ["Altis_Objects", [["Object1", {...}], ["Object2", {...}]]] }

3. The "Info" is save
  { "method": "set", "params": ["Altis_Objects", "Info", {...}]}

```

**Comment by IvanMMM** [comment]
Here you go :)
https://www.dropbox.com/s/1f9byhtxv896jt4/sock-rpc-stats.log?dl=0
Looks like this time all is working fine. When using restart-server-hint at least.

**Comment by micovery** [comment]
Thanks for log. Since it works through the `restart-server-hint` script , then it must be an issue with the invocation done through o_saveLoop, I'll investigate.

**Comment by IvanMMM** [comment]
Looks like that's the problem:

req: {"method":"set","params":["Altis_Objects",["Info.WarchestMoneyBLUFOR",0],["
Info.WarchestMoneyOPFOR",0]]}
req: {"method":"wipe","params":["Altis_Objects"]}

**Comment by micovery** [comment]
Fixed by f37bee6b2e9472c35e454ef11c23a1c2b052b096

save operations were in wrong order.
